commits.txt

```diff
@@ -1262,7 +1267,8 @@ https://github.com/servo/servo/pull/40491	(@mrobinson, #40491)	script: Rename `S
     # before fixing lints / formatting etc.
     # Testing: Manually tested with `./mach try`.
     # Fixes: #40475
-https://github.com/servo/servo/pull/40489	(@eerii, #40489)	tools: Remove wrapped JSON message in the DevTools parser (#40489)
++https://github.com/servo/servo/pull/40489	(@eerii, #40489)	tools: Remove wrapped JSON message in the DevTools parser (#40489)
```

Comment:
This isn't any kind of Servo-visible behaviour change.

Reply:
(didn’t end up mentioning this change)